### PR TITLE
Point to relevant dockerfile in readme for dependency installation

### PR DESCRIPTION
**Pull request by rsarky**
Removes duplication of dependencies in multiple places

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,14 +33,8 @@ Requirements
...
-On Ubuntu 18.10 as reference distro, those dependencies can easily installed
-with:
-```
-# apt install python3-sklearn python3-git python3-pygit2 python3-fuzzywuzzy
-	      python3-flaskext.wtf python3-pip python3-tqdm \
-              git procmail
-$ pip3 install --user flask-bootstrap flask-nav anytree
-```
+On Ubuntu 19.04 as reference distro, you can refer to this [Dockerfile](https://github.com/lfd/PaStA/blob/master/docker/pasta-skeleton.dockerfile)
+to download the required dependencies.
```
Comment (rralf) [inline README.md]: Can we reword it a bit and remove the "you". "Please refer to ...". For the link, can we somehow use markdown-magic to reference relatively without providing the full github link? I'd like to preserve a maximum line length of 80 characters. Thanks!

**Comment by rralf** [comment]
Thanks for the PR, find some comments in the patch. Ah, it doesn't matter for this patch, but PRs go against the next branch (the integration branch). Yes, I know, we lack documentation. ;-)

**Comment by rsarky** [comment]
Will incorporate the changes!
Aha, maybe I can add to the TODO, saying "Add a Contributing.md for new contributors". Or I can pick it up once I am familiar with the project

**Comment by rsarky** [comment]
Yeah, was just gonnna ask if you want me to squash the commits. Beat me to it!

**Comment by rsarky** [comment]
Done!

**Comment by rralf** [comment]
Aah, one last thing. Could you please add you Signed-Off? We use the same policy as they're used by Linux. Sorry, not mentioning that was my fault.

**Comment by rsarky** [comment]
Added a sign off on the commit.

**Comment by rralf** [comment]
Merged. Thanks! Will return to you soon.

**Pull request merged**